helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/HealthCheck.java

```diff
@@ -54,21 +55,24 @@ public enum HealthCheck {
   /**
    * Check if instance has error partitions
    */
-  HAS_ERROR_PARTITION,
-  /**
-   * Check if all resources hosted on the instance can still meet the min active replica
-   * constraint if this instance is shutdown
-   */
-  MIN_ACTIVE_REPLICA_CHECK_FAILED;
```

Comment:
This would break backward compatibility if the user depends the HealthCheck Enum. How about this? let's keep it there. But exclude enum MIN_ACTIVE_REPLICA_CHECK_FAILED before calling `getInstanceHealthStatus()` inside the method `performHelixOwnInstanceCheck()`
```
    List<HealthCheck> healthChecksToExecute = new ArrayList<>(HealthCheck.STOPPABLE_CHECK_LIST);
    healthChecksToExecute.removeAll(_skipStoppableHealthCheckList);
    healthChecksToExecute.remove(MIN_ACTIVE_REPLICA_CHECK_FAILED)
```

Reply:
Good point. Excluding the MIN_ACTIVE_REPLICA_CHECK_FAILED definitely works. We could also just set the MIN_ACTIVE_REPLICA_CHECK_FAILED switch case in `getInstanceHealthStatus()` to a no-op? What do you think is the clearest approach?